test/parallel/test-net-better-error-messages-port-hostname.js

```diff
@@ -3,12 +3,18 @@ const common = require('../common');
 const net = require('net');
 const assert = require('assert');
 
-const c = net.createConnection(common.PORT, '***');
+const s = net.createServer();
+s.listen(0);
+const port = s.address().port;
```

Comment:
I'm fine with `common.PORT` or a hard-coded number whether it be `8080`,  `0`, or whatever. Like I said, I have opinions, but not strong ones.